esp/services/esdl_svc_engine/esdl_binding.hpp

```diff
@@ -209,6 +214,27 @@ class EsdlServiceImpl : public CInterface, implements IEspService
     virtual bool subscribeServiceToDali() override {return false;}
     virtual bool attachServiceToDali() override {return false;}
     virtual bool detachServiceFromDali() override {return false;}
+
+private:
+    bool initMaskingEngineDirectory(const char* dir);
+    template <typename file_loader_t>
+    bool initMaskingEngineDirectory(const char* dir, const char* mask, file_loader_t loader)
```

Comment:
Can the implementation of this be moved to the cpp file with the other related functions?

Reply:
I can try. Since it's private, maybe the definition doesn't have to be in the header like most template definitions.